src/travel-details-screens/TripDetailsScreenComponent.tsx

```diff
@@ -133,9 +133,9 @@ export const TripDetailsScreenComponent = ({
             onPress={() => {
               analytics().logEvent('click_trip_purchase_button');
               onPressBuyTicket({
-                fareProductTypeConfig: singleTicketConfig,
-                fromPlace: tripTicketDetails.tariffZoneFrom,
-                toPlace: tripTicketDetails.tariffZoneTo,
+                fareProductTypeConfig: tripTicketDetails.fareProductTypeConfig,
```

Comment:
`onPressBuyTicket` does not require the config that is given to be of `type === 'single'`?

Reply:
I see that `getTicketInfoForBus` returns the config with type 'single', and `getTicketInfoForBoat` returns the 'boat-single', so I guess that works 👍

Reply:
Yes, the `getTicketInfoForBoat` and `getTicketInfoForBus` functions handles which config is passed to the `onPressBuyTicket` since these differs for single ticket bus and boat 👍